doc/release-notes/8611-DataCommons-related-notes.md

```diff
@@ -0,0 +1,78 @@
+
+## New JVM Options and DB Settings
+
+The following DB settings have been added:
+
+- `:LDNMessageHosts`
+- `: BasicGlobusToken`
+- `:GlobusEndpoint`
+- `:GlobusStores`
+- `:GlobusAppUrl`
+- `:S3ArchiverConfig`
+- `:S3ArchiverProfile`
+- `:DRSArchivalConfig`
+
+See the [Database Settings](https://guides.dataverse.org/en/5.11/installation/config.html#database-settings) section of the Guides for more information.
+
+## Notes for Developers and Integrators
+
+See the "Backward Incompatibilities" section below.
+
+## Backward Incompatibilities
+
+### OAI-ORE and Archival Bag Changes
+
+Earlier versions of the archival bags included the ingested (tab-separated-value) version of tabular files while providing the checksum of the original file (Issue #8849). This release fixes that by including the original file and its metadata in the archival bag. This means that archival bags created prior to this version do not include a way to validate ingested files. Further, it is likely that capabilities in development (i.e. as part of the (Dataverse Uplaoder)[https://github/org/GlobalDataverseCommunityConsortium/dataverse-uploader]) to allow re-creation of a dataset version from an archival bag will only be fully compatible with archival bags generated by a Dataverse instance at a release > v5.12. (Specifically, at a minimum, since only the ingested file is included in earlier archival bags, an upload via DVUploader would not result in the same original file/ingested version as in the original dataset.) Administrators should be aware that re-creating archival bags, i.e. via the new batch archiving API, may be advisable now and will be recommended at some point in the future (i.e. there will be a point where we will start versioning archival bags and will start maintaining backward compatibility for older versions as part of transitioning this from being an experimental capability).
+
+## Complete List of Changes
+
+
+
+## Installation
+
+If this is a new installation, please see our [Installation Guide](https://guides.dataverse.org/en/5.11/installation/). Please also contact us to get added to the [Dataverse Project Map](https://guides.dataverse.org/en/5.11/installation/config.html#putting-your-dataverse-installation-on-the-map-at-dataverse-org) if you have not done so already.
+
+## Upgrade Instructions
+
+8\. Re-export metadata files (OAI_ORE is affected by the PRs in this release note file) Optionally, for those using Dataverse's BagIt-based archiving, re-archive datasetversions archived using prior Dataverse versions. This will be recommended/required in a future release.
```

Comment:
```suggestion
8\. Re-export metadata files (OAI_ORE is affected by the PRs in these release notes). Optionally, for those using the Dataverse software's BagIt-based archiving, re-archive dataset versions archived using prior versions of the Dataverse software. This will be recommended/required in a future release.
```